### PR TITLE
Feature/pp9 8545

### PR DESCRIPTION
**Pull request by tapmapp**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -12,13 +11,17 @@ const routes: Routes = [
   },
   {
     path: 'content-picker',
-    component: ContentPickerComponent
+    loadChildren: './content-picker/content-picker.module#ContentPickerModule'
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/src/app/app-routing.module.ts]: Use Angular8 style lazy routing.

File: src/picturepark-sdk-v1-angular/tsconfig.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "outDir": "./dist/out-tsc",
     "sourceMap": true,
     "declaration": false,
-    "module": "esnext",
+    "module": "esNext",
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/tsconfig.json]: Needed?

Comment (tapmapp) [reply src/picturepark-sdk-v1-angular/tsconfig.json]: nope

File: src/picturepark-sdk-v1-angular/src/app/app-routing.module.ts (hunk 1 above)
Comment (tapmapp) [reply src/picturepark-sdk-v1-angular/src/app/app-routing.module.ts]: oh yes, one seg

**Pull request merged**